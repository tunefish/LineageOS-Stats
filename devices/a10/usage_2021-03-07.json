{
  "16.0": 39,
  "17.1": 691,
  "18.1": 119,
  "none": 198
}