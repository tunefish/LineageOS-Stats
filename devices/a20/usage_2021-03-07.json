{
  "17.1": 2416,
  "18.1": 74,
  "none": 24
}