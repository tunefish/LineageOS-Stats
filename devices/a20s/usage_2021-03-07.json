{
  "17.1": 2476,
  "none": 1
}