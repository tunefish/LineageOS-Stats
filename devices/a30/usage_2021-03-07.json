{
  "17.1": 29,
  "18.0": 681,
  "18.1": 287,
  "none": 4
}