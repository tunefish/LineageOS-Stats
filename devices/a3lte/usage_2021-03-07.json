{
  "15.1": 640,
  "16.0": 226,
  "17.1": 727
}