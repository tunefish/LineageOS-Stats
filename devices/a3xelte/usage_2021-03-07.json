{
  "14.1": 21,
  "15.1": 121,
  "16.0": 495,
  "17.1": 3971,
  "18.0": 3,
  "18.1": 14,
  "52.8": 1,
  "none": 2
}