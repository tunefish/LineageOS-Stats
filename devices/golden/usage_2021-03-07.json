{
  "13.0": 1128,
  "14.1": 3746,
  "15.1": 4,
  "17.1": 6,
  "none": 1
}